### ISSUE TITLE
Could you publish licensing information?

### ISSUE THREAD

**Issue by MW3000** [castiron/didh#13]
Could you publish which licenses apply to source code and text?

**Comment by mkgold** [comment]
Hi Martin,

Yes -- sorry, thought it was on the site but we can add that info. On our to-do list

Best,

Matt

**Comment by oledeuff** [comment]
Same question here.
Open source mode or not?

thanx

**Comment by alofro** [comment]
Hi!
same question again. What licenses apply to the source code?
Best regards!
ania

**Comment by zdavis** [comment]
Apologies for the delay on this, folks. I've added a CC-BY-NC-ND license to the texts directory, as the contents of the book are covered under that CC license, per the press' contributor contract.

I have also added a GPL license to the root of the project, per Matt's request.

I will warn you, that some parts of this project—especially the part that turns the chapter HTML files, which are extracted from the EPUB source file—are more proofs of concept than final, polished work. If you're interested in what we did here, I'd encourage you to keep an eye on the Manifold project that my team, Matt Gold, and the University of Minnesota Press are working on. Manifold aims to be a more general purpose reading, commenting, and annotation platform for EPUBs. It is also open source, and represents a more robust, considered approach to the problem of processing EPUBs. Check it out at https://github.com/ManifoldScholar/manifold. It's still far from done, super alpha, and lacks some essential documentation, but there's significant work that's been done around turning EPUB content into JSON data structures, which can in turn be rendered in a browser (via a frontend React application). If you're interested in this project, I'd encourage you to keep an eye on that one as well

Again, apologies for the delay on this!
